Microsoft.Toolkit.Win32/Microsoft.Toolkit.Sample.Wpf.XamlHost/WrappedButton.cs

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel;
+using System.Windows.Media;
+using Microsoft.Toolkit.Win32.UI.XamlHost;
+using Microsoft.Toolkit.Wpf.UI.XamlHost;
+
+namespace Microsoft.Toolkit.Sample.Wpf.XamlHost
+{
+    public class WrappedButton : WindowsXamlHostBase
+    {
+
+        #region Constructors and Initialization
+        public WrappedButton() : base()
+        {
+        }
+
+        protected override void OnInitialized(EventArgs e)
+        {
+            base.OnInitialized(e);
+
+            this.ChildInternal = UWPTypeFactory.CreateXamlContentByType("Windows.UI.Xaml.Controls.Button");
```

Comment:
In some places I've seen a ctor where the type is created and set to `ChildInternal`. Why is this done `OnInitialized` here?